### PR TITLE
feat: :sparkles: translate results.yml

### PR DESCRIPTION
**Pull request by b3nk3**
- user info
- locales
- features
- tools & methodologies
- sharing
- views
- tables

+ some random new line

### CODE REVIEW

File: results.yml [hunk 1]
```diff
@@ -185,143 +185,143 @@ translations:
...
+    t: Nyelv
   - key: user_info.locale.description
-    t: What language did respondents select to fill out the survey?
+    t: Milyen nyelvet választottak a válaszadók?
   - key: user_info.locale.note
     t: >
-      This data is collected automatically based on respondent's settings while taking the survey; 
-      Languages with fewer than 20 respondents not shown.
+      Nyelvi adatok gyűjtése automatikusan történik a válaszadók nyelvi beállításai alapján; 
+      Azon nyelvek melyeket kevesebb mint 20 válaszadó használt nem látható.
```
Comment (punkah) [inline results.yml]:
```suggestion
      Azon nyelvek, melyeket kevesebb, mint 20 válaszadó használt, nem láthatóak.
```

File: results.yml [hunk 2]
```diff
@@ -185,143 +185,143 @@ translations:
...
   - key: user_info.years_of_experience_by_salary
-    t: Years of Experience by Salary Range
+    t: Tapasztalat években, fizetések szerint rangsorolva
   - key: user_info.years_of_experience_by_salary.description
-    t: Years of experience by salary range.
+    t: Tapasztalat években, fizetések szerint rangsorolva.
 
   - key: user_info.yearly_salary_by_country
-    t: Salary By Country
+    t: Fizetés ország szerionti bontásban
```
Comment (punkah) [inline results.yml]:
```suggestion
    t: Fizetés ország szerinti bontásban
```

File: results.yml [hunk 3]
```diff
@@ -185,143 +185,143 @@ translations:
...
+    t: spanyol
   - key: options.locale.ru-RU
-    t: Russian
+    t: orosz
   - key: options.locale.fr-FR
-    t: French
+    t: francia
   - key: options.locale.zh-Hant
-    t: Chinese (Traditional)
+    t: tradicionális Kínai
```
Comment (punkah) [inline results.yml]:
```suggestion
    t: tradicionális kínai
```

File: results.yml [hunk 4]
```diff
@@ -185,143 +185,143 @@ translations:
...
 
   # general
   - key: tools.links
-    t: Links
+    t: Linkek
   - key: tools.github_link
     t: GitHub
   - key: tools.github_stars
-    t: stars
+    t: csillagok
```
Comment (punkah) [inline results.yml]:
```suggestion
    t: Csillagok
```

File: results.yml [hunk 5]
```diff
@@ -705,69 +705,69 @@ translations:
...
+    t: Táblázat
 
   ###########################################################################
   # Tables
   ###########################################################################
   - key: table.label
-    t: Label
+    t: Címke
   - key: table.item
     t: Item
```
Comment (punkah) [inline results.yml]:
```suggestion
    t: Elem
```

File: results.yml [hunk 6]
```diff
@@ -787,42 +787,42 @@ translations:
...
   - key: table.rankings_table
-    t: Rankings
+    t: Rangsorok
 
   - key: table.satisfaction_rank
-    t: Satisfaction Rank
+    t: Elégedettségi besorolás
   - key: table.interest_rank
-    t: Interest Rank
+    t: Érdeklődéso besorolás
```
Comment (punkah) [inline results.yml]:
```suggestion
    t: Érdeklődési besorolás
```

File: results.yml [hunk 7]
```diff
@@ -787,42 +787,42 @@ translations:
...
   - key: table.usage_rank
-    t: Usage Rank
+    t: Használati besorolás
   - key: table.awareness_rank
-    t: Awareness Rank
+    t: Ismertségi besorolás
 
   - key: table.would_not_use_percentage
-    t: Would not use %
+    t: Nem hásználnám többet %
```
Comment (punkah) [inline results.yml]:
```suggestion
    t: Nem használnám többet %
```

Comment (b3nk3) [reply results.yml]: haha, ugy tunik elojott az inner paloc :D

**Pull request merged**